include/veriblock/entities/vbk_merkle_path.hpp

```diff
@@ -46,6 +46,16 @@ struct VbkMerklePath {
   uint128 calculateMerkleRoot() const;
 };
 
+template <typename JsonValue>
+JsonValue ToJSON(const VbkMerklePath& mp) {
+  JsonValue obj = json::makeEmptyObject<JsonValue>();
+  json::putIntKV(obj, "treeIndex", mp.treeIndex);
+  json::putIntKV(obj, "index", mp.index);
+  json::putStringKV(obj, "subject", mp.subject.toHex());
```

Comment:
It was HexStr() in the MerklePath serialization above.

Reply:
These are same functions, so here it does not matter. It may matter with hashes, as AltBlock has vector<uint8_t> as hash, and others have Blob<N> as hash